ledger/src/blockstore_db.rs

```diff
@@ -742,7 +742,10 @@ impl Rocks {
     }
 
     fn write(&self, batch: RWriteBatch) -> Result<()> {
-        let is_perf_context_enabled = maybe_collect_perf_context();
+        let is_perf_context_enabled = maybe_collect_perf_context(
+            &self.3.perf_context_samples_in_1m,
+            &self.3.perf_write_counter,
+        );
```

Comment:
In a separate PR can you please name the fields of this struct:
https://github.com/solana-labs/solana/blob/3007f233f/ledger/src/blockstore_db.rs#L514-L520
instead of just a tuple, so that when reading this code `self.3` we can get some idea what `3` is.

Reply:
Sounds good!  Will do!

Reply:
#24761